src/core/crypto/singleKey.ts

```diff
@@ -199,7 +199,7 @@ export class AnySignature extends Signature {
   /**
    * Index of the underlying enum variant
    */
-  private readonly variant: AnySignatureVariant;
+  private readonly anySignatureVariant: AnySignatureVariant;
```

Comment:
Let's not rename this in this PR.  It's an unnecessary refactor.  To make changes like this should be in a separate commit and labeled appropriately

Reply:
I figure it's NBD because it is private.  I'd agree if it were not private, but it's completely internal as I understand.

Reply:
I just hope someone's not doing some weird serialization of this, but for now it should be fine.

These kinds of changes often lead to unexpected breaking changes.